README.md

```diff
@@ -9,31 +9,32 @@ How to use it:
 
 # BUILD
 ## Linux
+### Dependencies
+
+To install the required dependencies, run this command (Debian/Ubuntu):
 
-To build the editor on linux you need zlib and physfs development packages. If you use distribution based on Debian, you can install it using following command:
```

Comment:
This line is general for all distributions. Removing general information to leave only ubuntu/debian-specific information is not something we want.